packages/images-plugin/lib/finder.js

```diff
@@ -73,6 +73,11 @@ const DEFAULT_SETTINGS = {
 };
 
 export default class ImageElementFinder {
+  /**
+   *
+   * @param {import('@appium/types').Driver} driver
+   * @param {number} max
```

Comment:
is `max` the name or the description? if the name, it doesn't match, and if description, it's a bit laconic.